.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E261, E402, E501, W293, W504, F405, F403, E101, F841
+exclude = .git, .venv, build, cmake-build-*, **/temp/, *results/, requirements.txt
```

Comment:
update this to your virtual environment name (`.venv` was the name for my case), and the corresponding folders you want to ignore the flake8.